Directory.Packages.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+    <!-- Using multiple feeds isn't supported by Maestro: https://github.com/dotnet/arcade/issues/14155. -->
+    <NoWarn>$(NoWarn);NU1507</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.DotNet.RemoteExecutor" Version="$(MicrosoftDotNetRemoteExecutorVersion)" />
+    <PackageVersion Include="Microsoft.DotNet.XUnitExtensions" Version="$(MicrosoftDotNetXUnitExtensionsVersion)" />
+    <PackageVersion Include="System.Memory" Version="4.5.5" />
```

Comment:
So this will be imported for everyone? There used to be a comment in System.Buffers.csproj saying that it was a TODO to remove this dependency once we migrate System.Memory to this repo. Should we have such comment here too?

Reply:
Check out how NuGet CPM works: https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management

It's a feature that more and more repositories adopt in our stack. Runtime will follow at some point. These packages won't be imported by all projects. This file "only" specifies the centralized version.

Reply:
Thanks for the link, I read it and now I understand the basics and the meaning of having System.Memory indicated in the Directory.Packages.props file.

Follow-up question: When we migrate the source code of System.Memory to this repo, are we going to try to get all the projects in this repo that consume System.Memory to start consuming the migrated version instead?

Reply:
Yes, if System.Memory would be added to this repository, other libraries in this repository should reference its live version instead of the prebuilt on nuget.org.